doc/rst/platforms/cray.rst

```diff
@@ -535,9 +510,7 @@ When hugepages are used with the ugni comm layer, tasking layers
 cannot use guard pages for stack overflow detection.  Qthreads tasking
 can only use guard pages for stack overflow detection, so if ugni
 communications is combined with qthreads tasking, overflow detection is
-turned off completely.  Muxed tasking can use guard pages for stack
-overflow detection, but it can also drop back to synchronous overflow
-detection, as described below, with ``CHPL_COMM=ugni`` and hugepages.
+turned off completely.
```

Comment:
Not strictly true any more.  With comm=ugni, if you run without a craype-hugepages module loaded then you're in so-called "minimal registered heap" mode, most of memory including the stacks are on regular pages, and guard pages can be created (and are, other things permitting).  This is mentioned in the big chunk removed later, but it's kind of a passing reference.